chart/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+Triggermesh AWS event sources controller for knative has been installed.
+
+To view the controller logs:
+
+    kubectl logs -f deploy/{{ include "aws-event-sources.fullname" . }}-controller --namespace {{ .Release.Namespace }}
+
+Refer to https://github.com/triggermesh/aws-event-sources/tree/master/config/samples for examples that make use of the controller.
```

Comment:
URLs ?

Reply:
This are the chart notes. they will be printed in the terminal. Keeping them as urls will make them clickable from the terminal